include/zephyr/drivers/video.h

```diff
@@ -1377,6 +1338,15 @@ int video_transfer_buffer(const struct device *src, const struct device *sink,
  */
 #define VIDEO_PIX_FMT_GREY VIDEO_FOURCC('G', 'R', 'E', 'Y')
 
+
+/**
+ * @code{.unparsed}
+ *   0                   1                   2
+ * | 0000yyyy 0000Yyyy | 0000yyyy 0000Yyyy | 0000yyyy 0000Yyyy | ...
+ * @endcode
+ */
+#define VIDEO_PIX_FMT_Y4 VIDEO_FOURCC('Y', '0', '4', ' ')
+
```

Comment:
For this, it would be best to have a separate commit before the driver as it is not specific to this image sensor (at least... in theory).